services/movie.py

```diff
@@ -0,0 +1,45 @@
+from django.db.models import QuerySet
+
+from db.models import Movie, Genre, Actor
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
+    if not genres_ids and not actors_ids:
+        return Movie.objects.all()
+
+    movies = Movie.objects.all()
+
+    if genres_ids:
+        movies = movies.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        movies = movies.filter(actors__id__in=actors_ids)
+    return movies.distinct()
+
+
+def get_movie_by_id(movie_id: int) -> Movie | None:
+    try:
+        return Movie.objects.get(id=movie_id)
+    except Movie.DoesNotExist:
+        print("no movie")
+        return
+
+
+def create_movie(movie_title: str,
+                 movie_description: str,
+                 genres_ids: list[int] = None,
+                 actors_ids: list[int] = None) -> Movie:
+
+    movie = Movie.objects.create(
+        title=movie_title,
+        description=movie_description
+    )
+

```

Comment:
you can use method set, that don't make unnecessary requests(movie.genres.set(genres_ids))